raiden/tests/conftest.py

```diff
@@ -47,6 +31,23 @@
 from raiden.utils.debugging import enable_gevent_monitoring_signal
 from raiden.utils.ethereum_clients import is_supported_client
 
+monkey.patch_all(subprocess=False, thread=False)  # isort:skip # noqa
+
+import signal  # isort:skip # noqa
+import pytest  # isort:skip
+
+# Execute these before the other imports because rewrites can't work after the
```

Comment:
Hmmm these should, also according to the comment be before all the other imports.